Shared/Samples/Set viewpoint rotation/SetViewpointRotationView.swift

```diff
@@ -39,8 +39,7 @@ struct SetViewpointRotationView: View {
                 }
                 .overlay(alignment: .topTrailing) {
                     Compass(
-                        viewpointRotation: $rotation,
-                        autoHide: false
+                        viewpointRotation: $rotation
                     )
                     .padding()
```

Comment:
The old `autoHide` argument is now a modifier, but the meaning has flipped, it's now whether you want to disable the auto-hide behavior, which for this case would be true (in order to match previous behavior).
```
                    Compass(viewpointRotation: $rotation)
                        .autoHideDisabled(true)
                        .padding()
```